docs/static/monitoring/monitoring-ea.asciidoc

```diff
@@ -15,6 +15,13 @@ from a central location, or you can run {fleet-guide}/install-standalone-elastic
 
 Complete these steps as you prepare to collect and ship monitoring data for stack monitoring:
 
+[[set-up-monitoring-ea]]
+.Set up {es} monitoring
+[%collapsible]
+====
+include::monitoring-prereq-setup-es.asciidoc[]
+====
+
```

Comment:
review note: moved this step to first place as it is prerequisite.

Reply:
@mashhurs, the "review notes" you added throughout this PR are quite helpful. Thank you for taking the time.